mapstory/settings/base.py

```diff
@@ -356,4 +349,11 @@
             "handlers": ["console"], "level": "ERROR", },
 
         },
-    }
\ No newline at end of file
+    }
+
+if USE_AWS_S3:
+    AWS_S3_CUSTOM_DOMAIN = '%s.s3.amazonaws.com' % AWS_STORAGE_BUCKET_NAME
+    STATIC_URL = "https://%s/" % AWS_S3_CUSTOM_DOMAIN
+    STATICFILES_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
+
+REMOTE_CONTENT_URL = STATIC_URL + 'assets'
```

Comment:
I think it was added to pull images from the mapstory-assets report but do we still need this?